plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClientModule.java

```diff
@@ -38,6 +39,7 @@ public class PostgreSqlClientModule
     public void setup(Binder binder)
     {
         binder.bind(JdbcClient.class).annotatedWith(ForBaseJdbc.class).to(PostgreSqlClient.class).in(Scopes.SINGLETON);
+        newOptionalBinder(binder, TimestampTimeZoneDomain.class).setBinding().toInstance(TimestampTimeZoneDomain.UTC_ONLY);
```

Comment:
what verifies that the declaration is correct?

Reply:
i don't know

Reply:
added a test and verified it with Oracle connector, which preserves zone information